pkg/query/config.go

```diff
@@ -0,0 +1,53 @@
+package query
+
+import (
+	"github.com/pkg/errors"
+	"github.com/thanos-io/thanos/pkg/httpconfig"
+	"gopkg.in/yaml.v2"
+)
+
+type EndpointConfig struct {
+	Endpoints   []string                  `yaml:"addresses"`
+	EndpointsSD []httpconfig.FileSDConfig `yaml:"file_sd_configs"`
```

Comment:
Does this need to be a slice? Also, why not `file.SDConfig`? 🙂

Reply:
was getting errors actually..
*file.SDConfig is correct :+1: